pkg/controller/vpcpeering/peering.go

```diff
@@ -561,3 +569,33 @@ func (e *external) deleteVPCPeeringConnection(ctx context.Context, cr *svcapityp
 
 	return nil
 }
+
+func (e *external) checkRoutes(ctx context.Context, cr *svcapitypes.VPCPeeringConnection) (int, error) {
```

Comment:
nit: countRoutes may describe the function better :)

Reply:
Sure thing.